src/themes/default/_containers.scss

```diff
@@ -10,7 +10,7 @@
         flex-grow: 1;
         height: 90%;
         margin: 0;
-        padding: 0 $rhythm * 2;
+        padding: 10px $rhythm * 2 10px;
```

Comment:
this could just be `10px $rhythm * 2`
padding has 4 argument styles 
https://developer.mozilla.org/en-US/docs/Web/CSS/padding
if you want top and bottom and left and right to be the same you just need 2 values. `<top/botom> <right/left>`

also why 10px? where are you getting that value from?

Reply:
Ok, I didn't know that you didn't have to repeat padding for top and bottom if their both the same. Noted. It was a guesstimation on what looked good in my opinion.